### DISCUSSION TITLE
GitHub CI-CD for Xamarin doesn't able to find Android NDK during build

### DISCUSSION THREAD

**Discussion by softsan** [community/community#28403] [Actions]
Hello,

I have problem building Xamarin.Android on Github Action. It was working fine Yesterday. but today it gives following error and couldn't build further:

`/Library/Frameworks/Mono.framework/External/xbuild/Xamarin/Android/Xamarin.Android.Common.targets(1954,3): error XA5104: Could not locate the Android NDK. Please make sure the Android NDK is installed in the Android SDK Manager, or if using a custom NDK path, please ensure the $(AndroidNdkDirectory) MSBuild property is set to the custom path.`

I haven't updated anything regards to the android.
In a success build it does find the correct NDK path ie.: `Android NDK: /Users/runner/Library/Android/sdk/ndk/21.4.7075529/`
but in failed build (which started happening today) the Android NDK path is just blank.

Any Idea?

**Comment by susanrama** [comment]
Android NDK r21 was replaced with r25. See the following thread for mitigation guide: 
https://github.com/actions/virtual-environments/issues/5930
